CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Contributing guidelines
+
+## How to become a contributor and submit your own code
+
+See [this doc](https://github.com/kubernetes/community/blob/master/sig-cli/CONTRIBUTING.md) for how to contribute code.
+
+Code in this repo is synced from the k8s.io/kubernetes repo.  Code should not be directly contributed to this repo.
```

Comment:
Currently (q2/2017), this repository exists for issue tracking and code mirroring only, and is not accepting code contributions.   It will open for code contributions when migration of kubectl code from the [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) repository to this repository has been completed.